Editor config document (#37866)
kubernetes/website: content/en/docs/contribute/style/style-guide.md

These steps ... | These simple steps ...
{{< /table >}}

### EditorConfig

> EditorConfig is useful for maintaining consistant coding style across all the editors , this helps the developers using different editors (like vim , emacs , atom , vscode etc) would be able to contribute to the project with the same style.usually for doing this an .editorconfig file is used. 

Reviewer: small typo consistant -> `consistent` ?

> to check out the .editorconfig click [here](https://github.com/kubernetes/website/blob/main/.editorconfig) 

Reviewer 2: These sentences don't flow well. Please check.

Reviewer 3: Maybe a reword to:

```suggestion
The Kubernetes project maintains an EditorConfig file that sets common style preferences in text editors
such as VS Code.  You can use this file if you want to improve how consistent your contributions are with
the rest of the project. To download the file, refer to
[`.editorconfig`](https://github.com/kubernetes/website/blob/main/.editorconfig).
```

Reviewer 4: I like this phrasing better @shannonxtreme, but it may be odd to ask folks to download the file since it's a part of the k/website repo.

It may be better to refer to the file's location. Or mention the .editorconfig file in the body of the paragraph with the link to it?

Reviewer 3: Good point. To view the file, refer to .editorconfig in the repository root?

Reviewer 5: Hi @Karthik-PM . What do you think about incorporating the suggested edits?

These steps ... | These simple steps ...
{{< /table >}}

### EditorConfig
The Kubernetes project maintains an EditorConfig file that sets common style preferences in text editors

> such as VS Code.  You can use this file if you want to improve how consistent your contributions are with

Reviewer 6: ```suggestion
such as VS Code. You can use this file if you want to ensure that your contributions are consistent with
```

> the rest of the project. To view the file, refer to

Reviewer 6: ```suggestion
the project's [Documentation Style Guide](docs/contribute/style/style-guide/). To view the file, refer to
```

> [`.editorconfig`](https://github.com/kubernetes/website/blob/main/.editorconfig) in the repository root.

Reviewer 5: nit: Add a new line after the last sentence

These steps ... | These simple steps ...
{{< /table >}}

### EditorConfig

> For maintaing a constant coding style we would need you to maintain a  .editorconfig file 

Reviewer 7: As this is the style guide itself, we **must** [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).

Also, I don't think the Kubernetes project is actually insisting that contributors maintain a `.editorconfig`. Maybe what we're saying is that the repository includes a `.editorconfig` and that this may help people to follow the style guide.

so that the developers using different editors (like vim , emacs , atom , vscode etc) for the same project can contribute with the same style.

the .editorconfig used here is 
```
   [*]
   end_of_line = lf
   insert_final_newline = false
   charset = utf-8
   max_line_length = 80
   trim_trailing_whitespace = true

   [*.md]
   trim_trailing_whitespace = false

   [*.{css,html,js,json,sass,md,mmark,toml,yaml}]
   indent_style = space
   indent_size = 2

   [*.{sh}]
   indent_style = space
   indent_size = 4

   [Makefile]

>    indent_style = tab

Reviewer 7: This is likely to go stale; maybe we can hyperlink to the file instead?

Author: @sftm i have added the changes , can you look into it?
